PosteAPI/migrations/0002_remove_folder_shared_users_folderpermission.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.2.5 on 2023-10-15 02:14
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
For now, while still in development, it may be easier for us to simply delete and recreate migrations than build on top. Once in a working environment where we want to maintain data integrity the creation of new migrations is wise, but as we are still establishing a baseline for our models, a single initial migration may suffice (and avoid back-and-forth migration bloat)